addons/stock/views/stock_picking_views.xml

```diff
@@ -45,6 +45,7 @@
                                 <field name="warehouse_id" groups="stock.group_stock_multi_warehouses" force_save="1"/>
                                 <field name="print_label" attrs="{'invisible': [('code', '!=', 'internal')]}"/>
                                 <field name="reservation_method" attrs="{'invisible': [('code', '=', 'incoming')]}" widget="radio"/>
+                                <field name="auto_show_reception_report" attrs="{'invisible': [('code', '=', 'outgoing')]}" groups="stock.group_reception_report"/>
```

Comment:
internal picking type will have the button too ?

Reply:
For now, yes. Otherwise we can't do end-to-end multi-step allocations (i.e. reception -> internal -> outgoing manual MTO). Also for more dynamic MTO setting/removing flexibility/power :muscle: